ext/opcache/ZendAccelerator.h

```diff
@@ -223,8 +223,7 @@ typedef struct _zend_accel_globals {
 	const zend_op          *cache_opline;
 	zend_persistent_script *cache_persistent_script;
 	/* preallocated buffer for keys */
-	zend_string             key;
-	char                    _key[MAXPATHLEN * 8];
+	zend_string            *key;
```

Comment:
Oops. I forgot this is an ABI break. We'll need to target `master` I think.

Reply:
Hmm, is this an ABI break? After all, ZendAccelerator.h is not published (aka. exported).

Reply:
Well, it is an ABI break, since the layout of the struct is now different. The question is whether this is relevant. Certainly safer to target `master` only.

Reply:
I meant I didn't know this wasn't a published header. So it should be fine.

Reply:
Some may access the struct without using a header. cc @bwoebi, @realFlowControl

Reply:
Thanks for the ping @cmb69, I am good with the change.